resource/graph_node.go

```diff
@@ -32,6 +33,7 @@ type GraphNode struct {
 	currentModel              Model
 	config                    Config
 	needsReconfigure          bool
+	lastReconfigured          *time.Time
```

Comment:
I'm guessing the choice to use a pointer here was to avoid a default value of "Jan 1, 1970" from being missed and sent back as part of a response?

Reply:
Sort of; it's to allow a proper value for the field for uninitialized or unconfigured nodes. Only configured nodes really have a sense of `lastReconfigured`. I thought we could be more expressive in the backend (the `lastReconfigured` field is not defined as `optional` in the proto).